lib/devos/default.nix

```
@@ -7,6 +7,15 @@
       config = { allowUnfree = true; };
     };
 
+  mkBuild = config: module:
```

```suggestion
  mkBuild = config: buildModule:
```

Just to make it a tiny bit more obvious tht we are talking about a specific type of module, not "any" module.

---

done in aa17170